### PR TITLE
Hexagon

### PR DESCRIPTION
**Pull request by cristeahub**

### CODE REVIEW

**Comment by stianjensen** [comment]
Please rebase.

**Comment by stianjensen** [comment]
Also, does text actually work?

File: src/WallLayer.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    innerHex.rotation.y = 1 / 6 * Math.PI;
+  }
+
+  this.scene.add(hex);
+  this.scene.add(innerHex);
+
+  return { outer: hex, inner: innerHex };
+}
+
+WallLayer.prototype.randomNum = function(threshhold) {
```
Comment (iver56) [inline src/WallLayer.js]: _threshold_

Comment (cristeahub) [inline src/WallLayer.js]: Stian: Yes and I'll rebase it :)

**Comment by cristeahub** [comment]
Plingaling pooo pooo :baby:

**Comment by cristeahub** [comment]
plingaling :tomato:

**Comment by stianjensen** [comment]
I don't see the content of the hexagon that's supposed to say 'presents' and the one that says 'Inakuwa oasis'.

**Comment by peterhgombos** [comment]
I do, but there's some noise when running the intro. And can we please, for the love of god, use another font? Most clichéd African font ever!

**Comment by stianjensen** [comment]
I think the font stays – but we're changing the color to white, that's much better.

**Comment by cristeahub** [comment]
Yes, the font color made the font look pretty bad, it's been fixed now, and it looks a lot cleaner :)

also: ping :package:

**Pull request merged**